tests/providers/openlineage/utils/test_selective_enable.py

```diff
@@ -0,0 +1,72 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from airflow.decorators import dag, task
+from airflow.models import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.providers.openlineage.utils.selective_enable import (
+    DISABLE_OL_PARAM,
+    ENABLE_OL_PARAM,
+    ENABLE_OL_PARAM_NAME,
+    disable_lineage,
+    enable_lineage,
+)
+
+
+class TestOpenLineageSelectiveEnable:
```

Comment:
I think we should also add some test when there are different settings for DAG and Task. F.e. when enabled on DAG level but disabled for one of tasks.

Reply:
Added test case.